src/app/features/home-feature/components/templates/login-template/login-template.component.html

```diff
@@ -1,12 +1,15 @@
-<div class="login-template-container">
+<form class="login-template-container" [formGroup]="loginForm" (ngSubmit)="onSubmit()">
   <sk-card class="card">
     <h1 class="title">Logg inn</h1>
-    <sk-input-email class="email" [placeholder]="'E-post'" (valueChange)="onEmailChanged($event)"></sk-input-email>
-    <sk-input-password class="password" [placeholder]="'Passord'" (valueChange)="onPasswordChanged($event)"></sk-input-password>
-    <sk-button class="login-button" [variant]="'primary'" (clicked)="onLogin()"> Logg inn </sk-button>
+    <sk-input-text class="email" [type]="'email'" [placeholder]="'E-post'" formControlName="email"></sk-input-text>
```

Comment:
Are the dynamic attributes necessary?

```suggestion
    <sk-input-text class="email" type="email" placeholder="E-post" formControlName="email"></sk-input-text>
```

Reply:
It is best practice since the compiler won't fail on non-dynamic attributes.
Example: `my-attribute` does not exist on component, [my-attribute] will fail, my-attribute won't.

Also the IDE will give warnings on things like [type]="'this-is-not-valid'", but not type="this-is-not-valid".